codalab/worker/local_run/local_run_state.py

```diff
@@ -209,30 +212,42 @@ def _transition_from_PREPARING(self, run_state):
 
         # All dependencies ready! Set up directories, symlinks and container. Start container.
         # 1) Set up a directory to store the bundle.
-        remove_path(run_state.bundle_path)
-        os.mkdir(run_state.bundle_path)
+        if self.shared_file_system:
+            try:
+                self._wait_for_bundle_folder(run_state)
+            except self.NoBundleFolderException as ex:
+                message = 'Bundle folder unavailable: %s' % str(ex)
```

Comment:
I assume the user will see this failure message. Is it actionable? (i.e. user should delete and retry) If so, specify that in the message as well

Reply:
Yeah, although something along the lines of 'ensure your bundle store is properly mounted to your worker host machine' would be better.

Reply:
hmm. On one hand, if the worker is user-owned, 'ensure your bundle store is properly mounted to your worker host machine' would be a helpful message. On the other, if the user doesn't own the worker, that same message would be less helpful.

Reply:
Yeah I guess that + contact the administrator should cover both cases